help/destinations/catalog/crm/outreach.md

```diff
@@ -0,0 +1,217 @@
+![[!DNL Outreach] custom fields defaults](../../assets/catalog/crm/outreach/outreach-custom-fields.png)
+
+[!DNL Outreach] settings page for prospects displaying custom fields with *user-friendly* labels matching to the segment names. You can view the segment status on the prospect page against these labels.
+![[!DNL Outreach] custom fields with labels](../../assets/catalog/crm/outreach/outreach-custom-field-labels.png)
+
+> [!NOTE]
+>
+> Label names are for ease of identification only, they are not used when updating prospects.
+
+## Supported identities {#supported-identities}
+
+[!DNL Outreach] supports update of identities described in the table below. Learn more about [identities](/help/identity-service/namespaces.md).
+
+|Target Identity|Description|Considerations|
+|---|---|---|
+| `OutreachId` | <ul><li>[!DNL Outreach] identifier, this is a numeric value corresponding to the prospect profile.</li><li>The ID must match the ID within the [!DNL Outreach] URL for the prospect being updated.</li><li>Refer to the [[!DNL Outreach] documentation](https://api.outreach.io/api/v2/docs#update-an-existing-resource) for further details.</li></ul>|Mandatory |
+
+## Export type and frequency {#export-type-frequency}
+
+Refer to the table below for information about the destination export type and frequency.
+
+| Item | Type | Notes |
+---------|----------|---------|
+| Export type | **[!UICONTROL Profile-based]** | <ul><li>You are exporting all members of a segment, together with the desired schema fields *(for example: email address, phone number, last name)*, according to your field mapping.</li><li> Platform segment statuses are exported to [!DNL Outreach] by specifying their corresponding custom field attribute in [!DNL Outreach], within the **[!UICONTROL Activate Destination]** > **[!UICONTROL Schedule segment export]** > **[!UICONTROL Mapping ID]** field.</li></ul> |
+| Export frequency | **[!UICONTROL Streaming]** | <ul><li>Streaming destinations are "always on" API-based connections. As soon as a profile is updated in Experience Platform based on segment evaluation, the connector sends the update downstream to the destination platform. Read more about [streaming destinations](/help/destinations/destination-types.md#streaming-destinations).</li></ul>|
+
+{style="table-layout:auto"}
+
+## Connect to the destination {#connect}
+
+> [!IMPORTANT]
+> 
+> To connect to the destination, you need the **[!UICONTROL Manage Destinations]** [access control permission](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+
+To connect to this destination, follow the steps described in the [destination configuration tutorial](../../ui/connect-destination.md). In the configure destination workflow, fill in the fields listed in the two sections below.
+
+![Catalog](../../assets/catalog/crm/outreach/catalog.png)
```

Comment:
This screenshot looks pretty cluttered. I would remove it completely, since this step is standard across all destinations and is described in the article you already linked to.

Reply:
If we remove it the user might not specifically choose the connector under the CRM category. Have updated with fewer highlights and no numbering (which is why I had avoided it in the first place).

Reply:
Have removed the screenshot.